### PR TITLE
MT2 updates and validation

### PR DESCRIPTION
**Pull request by mariadalfonso**
Things of general interest:
1) write output in the CERN EOS
2) added private T2tt-T2bb-T2qq sample 
3) added class to apply recoil corrections
4) speed up a bit the IsoTrackModule
5) commented the default tkMET and QG since are time consuming

MT2:
1) added validation script
2) updated mindPHI (eta range) and Zinvp4
3) added friend tree for kinematic variables 
4) cleaned up MT2 calculation file

### CODE REVIEW

File: CMGTools/TTHAnalysis/python/analyzers/ntupleTypes.py [hunk 1]
```diff
@@ -208,12 +208,13 @@
     #NTupleVariable("PuId_cut_based", lambda x : x.puId("cut-based"), int,    mcOnly=False, help="puId cut-based: returns an integeger containing 3 bits, one for each working point (loose-bit2, medium-bit1, tight-bit0)"),
     NTupleVariable("id",    lambda x : x.jetID("POG_PFID") , int, mcOnly=False,help="POG Loose jet ID"),
     # QG variables:
-    NTupleVariable("ptd",   lambda x : x.ptd, float, mcOnly=False,help="QG input variable: ptD"),
-    NTupleVariable("axis2",   lambda x : x.axis2 , float, mcOnly=False,help="QG input variable: axis2"),
-    NTupleVariable("mult",   lambda x : x.mult , int, mcOnly=False,help="QG input variable: total multiplicity"),
-    NTupleVariable("qgl",   lambda x : x.qgl , float, mcOnly=False,help="QG Likelihood"),
-    NTupleVariable("partonId", lambda x : x.partonId, int,     mcOnly=True, help="parton flavour (manually matching to status 23 particles)"),
-    NTupleVariable("partonMotherId", lambda x : x.partonMotherId, int,     mcOnly=True, help="parton flavour (manually matching to status 23 particles)"),
+    NTupleVariable("svRedPt", lambda x : getattr(x, 'ivfRedPt', 0), help="pT of associated SV, removing the lepton track"),
```
Comment (gpetruc) [inline CMGTools/TTHAnalysis/python/analyzers/ntupleTypes.py]: why svRedPt is under QG variables?

Comment (pandolf) [inline CMGTools/TTHAnalysis/python/analyzers/ntupleTypes.py]: i dont know what that is, i didnt put it there.

f

**Comment by mariadalfonso** [comment]
Ciao Giovanni.
I fixed the duplicated svRedPt

Maria

**Comment by mariadalfonso** [comment]
Slimmed IsoTrack format to minimal for timing improvements

**Comment by gpetruc** [comment]
Not merging yet as @mariadalfonso told me she wants to fix another thing in the isolated track veto

**Pull request merged**